app/views/pages/home.html.erb

```diff
@@ -0,0 +1,5 @@
+<header class="header">
+    <h1>
+        <span>E-Budget</span>
+    </h1>
+</header>
```

Comment:
- [ ] The project is a mobile application and as such should be designed for mobile. Please ensure that your design is close to the original design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/189240708-3cafed48-35c9-4eb6-9d66-96e6b6189169.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/189240649-0f3dd7e8-f9aa-42f4-a532-cf8e2913bb07.png)